fastlane/lanes/build.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+platform :mac do
+  lane :build_and_upload_app_store do |options|
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/simplenote-macos/blob/34ff326c579801ec45af5c8956c0014425743887/fastlane/lanes/build.rb#L4"></span>Lint/UnusedBlockArgument: Unused block argument - <code>options</code>. You can omit the argument if you don't care about it.</td>
    </tr>
  </tbody>
</table>